### DISCUSSION TITLE
Syncing GitHub on Ubuntu and Windows dual boot

### DISCUSSION THREAD

**Discussion by andylatham82** [community/community#24094] [Apps, API and Webhooks]
<p>Hi!</p>
<p>I recently installed Ubuntu onto my PC as a dual boot with Windows 10. I have been creating a game on Windows and using GitHub Desktop for syncing with GitHub. I have learned how to use Git on Linux and how to push and pull from GitHub from the terminal.</p>
<p>Now my game project files are stored locally on a hard drive that is accessible to both Windows and Ubuntu. I can happily work on my game in either OS. However I haven’t yet worked out what the process is syncing up with GitHub. To be safe I have just been pushing updates from Windows via GitHub Desktop.</p>
<p>Am I safe to work on the same files in Ubuntu, commit and push them to GitHub via the terminal? When I log back into GitHub Desktop on Windows will it tell me that everything is synced up? I just don’t want to screw anything up!</p>
<p>Many thanks!</p>

**Comment by andylatham82** [comment]
<p>Further to this, I have just tried doing a <code>git remote update</code> followed by <code>git status</code> on my game directory in Ubuntu and it thinks that the files on my hard drive are different than those on GitHub (they all show up red in the terminal). I could try committing them and pushing them to GitHub, but I’m not sure if GitHub Desktop in Windows will then think things are out of date and I’ll end up in a cycle of committing and updating without anything actually having changed!</p>

**Comment by andylatham82** [comment, marked as answer]
<p>Never mind, I figured it out!</p>
<p>I just did a git add all, commit and push in Ubuntu and thankfully GitHub Desktop in Windows was able to see that everything was synced up.</p>

**Comment by dimaslanjaka** [comment]
try re-index pack when switched to other OS https://git-scm.com/docs/git-index-pack to improve `git status` speed, and this will not change any staged/untracked/modified files. just for sync between git packs
